src/stack.js

```diff
@@ -1,5 +1,26 @@
 'use strict'
 
 export default class Stack {
-  // your code here
+
+  constructor() {
+    this.stack = []
```

Comment:
Hi Punit - Thanks for the review. I'm not quite ready for nodes, but I understand the recommendation.